src/main/java/com/limechain/beefy/dto/PayloadElement.java

```diff
@@ -1,8 +1,12 @@
 package com.limechain.beefy.dto;
 
+import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Data
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
```suggestion
@Value
```

and remove `private` modifiers.